src/dialogs/more-info/controls/more-info-media_player.js

```diff
@@ -255,6 +280,27 @@ class MoreInfoMediaPlayer extends LocalizeMixin(EventsMixin(PolymerElement)) {
     this.playerObj.selectSource(sourceInput);
   }
 
+  handleChannelChanged(channelIndex, channelIndexOld) {
+    // Selected Option will transition to '' before transitioning to new value
+    if (!this.playerObj
+        || this.playerObj.channelList === undefined
+        || channelIndex < 0
+        || channelIndex >= this.playerObj.channelList
```

Comment:
Comparing number to array? And I don't think `channelIndex` can ever actually be larger than the size of `channelList` can it? It either comes from the `indexOf` call above or selection in the dropdown

Reply:
I've just followed the way handleSourceChange was written. I supposed there was a reason for this (mayabe if the channel list changes and a channel is added?). If not, I can rewrite both functions.

Reply:
Hello, I've just corrected the length and added a comment to explain why the test is there